omnigibson/robots/fetch.py

```diff
@@ -271,6 +275,10 @@ def _initialize(self):
             for joint in self.finger_joints[arm]:
                 if joint.joint_type != JointType.JOINT_FIXED:
                     joint.friction = 500
+        
+        # Keep track of tucked chassis footprint
+        self.tuck()
```

Comment:
After thinking about this twice, I think we should remove the special casing for fetch and tiago. Essentially these two robots allow users to specify the reset_joint_pos to be tucked or untucked. The aabb extent should probably respect that argument accordingly.

Reply:
@ChengshuLi correct me if I'm wrong, but I think the proposal is to not need to reset/tuck/etx the robot at all, just use the configuration at the end of initialize. This should already be the reset pose.

What happens on a robot whose joints might be saved in the JSON file though? Please make sure load_state doesnt get called before you record this because it will set the joints arbitrarily